pkg/app/metadata.json

```diff
@@ -0,0 +1,965 @@
+      "apis": ["https://developer.fastly.com/reference/api/acls/acl-entry/#create-acl-entry"]
+    },
+    "delete": {
+      "examples": [{
+        "cmd": "fastly acl-entry delete --acl-id 50CCynNwD3BjaGJjMTFB1j --id 4DiuYrv9nVoa4HFmQmujT1",
+        "title": "Delete an ACL entry from a specified ACL"
+      }],
+      "apis": ["https://developer.fastly.com/reference/api/acls/acl-entry/#delete-acl-entry"]
+    },
+    "describe": {
+      "examples": [{
+        "cmd": "fastly acl-entry describe --acl-id 50CCynNwD3BjaGJjMTFB1j --id 4bgjM6X1gb3670gJMEqdKX",
+        "title": "Retrieve a single ACL entry"
+      }],
+      "apis": ["https://developer.fastly.com/reference/api/acls/acl-entry/#get-acl-entry"]
+    },
+    "list": {
+      "examples": [{
+        "cmd": "fastly acl-entry list --acl-id 50CCynNwD3BjaGJjMTFB1j",
+        "title": "List ACLs from a specified ACL"
+      }],
+      "apis": ["https://developer.fastly.com/reference/api/acls/acl-entry/#list-acl-entries"]
+    },
+    "update": {
+      "examples": [{
+        "cmd": "fastly acl-entry update --acl-id 50CCynNwD3BjaGJjMTFB1j --id 4bgjM6X1gb3670gJMEqdKX --negated",
+        "title": "Update a specific ACL entry"
+      },
+      {
+        "cmd": "fastly acl-entry update --acl-id 50CCynNwD3BjaGJjMTFB1j --file ./batch.json",
+        "description": "Update multiple ACL entries using JSON batch file. Refer to https://developer.fastly.com/reference/api/acls/acl-entry/#bulk-update-acl-entries for example structure.",
+        "title": "Update multiple ACL entries using a local file"
+      },
+      {
+        "cmd": "fastly acl-entry update --acl-id 50CCynNwD3BjaGJjMTFB1j --file \"$(< batch.json)\"",
+        "description": "Update multiple ACL entries using JSON batch file content passed in using shell command substitution. Refer to https://developer.fastly.com/reference/api/acls/acl-entry/#bulk-update-acl-entries for example structure.",
+        "title": "Update multiple ACL entries using command substitution"
+      }],
+      "apis": [
+        "https://developer.fastly.com/reference/api/acls/acl-entry/#bulk-update-acl-entries",
+        "https://developer.fastly.com/reference/api/acls/acl-entry/#update-acl-entry"
+      ]
+    }
+  },
+  "auth-token": {
+    "create": {
+      "apis": ["https://developer.fastly.com/reference/api/auth/#create-token"]
+    },
+    "delete": {
+      "apis": [
+        "https://developer.fastly.com/reference/api/auth/#revoke-token-current",
+        "https://developer.fastly.com/reference/api/auth/#bulk-revoke-tokens",
+        "https://developer.fastly.com/reference/api/auth/#revoke-token"
+      ]
+    },
+    "describe": {
+      "apis": ["https://developer.fastly.com/reference/api/auth/#get-token-current"]
+    },
+    "list": {
+      "apis": [
+        "https://developer.fastly.com/reference/api/auth/#list-tokens-customer",
+        "https://developer.fastly.com/reference/api/auth/#list-tokens-user"
+      ]
+    }
+  },
+  "backend": {
```

Comment:
Mostly basic examples. The ‘create’ example felt more like an excuse to write a description that’s less of a description and more a note about the host behaviour (which is possibly an abuse of the description field).